functional_tests/steps/bundles.py

```diff
@@ -121,3 +128,267 @@ def the_user_can_see_the_bundle_details_with_creator(context: Context) -> None:
 def bundle_inspect_show(context: Context) -> None:
     expect(context.page.get_by_text("Created by")).to_be_visible()
     expect(context.page.get_by_text(context.bundle_creator.get_username())).not_to_be_visible()
+
+
+# Bundles UI Data Setup
+
+
+@given("there is a {user_role} user")
+def a_user_exists_by_role(context: Context, user_role: str) -> None:
+    if not hasattr(context, "users"):
+        context.users = []
+
+    context.users.append({user_role: create_user(user_role)})
+
+
+@given("there are {no_statistical_analysis} Statistical Analysis pages")
+def multiple_statistical_analysis(context: Context, no_statistical_analysis: str) -> None:
+    context.statistical_article_pages = []
+    context.article_series_page = ArticleSeriesPageFactory(title="PSF")
+    if no_statistical_analysis.isdigit():
```

Comment:
Do we need to check if `no_statistical_analysis` is a digit? It seems to me that it should always be a digit.

Reply:
Could you show me in what scenario that would not be an integer?

Reply:
There is not a scenario where this would not be an integer just instinct where input is so precarious the same with json.loads